### PR TITLE
Improve handling styles for errors/warnings in localized inputs and fields

### LINKED ISSUE #2730: `LocalizedTextField` should allow passing `errorsByLanguage` as React nodes and not only as strings
**Describe the bug**
I'm currently using `LocalizedTextField` which renders multiple inputs for all the languages, I want to show error under each language and not a global error under the whole field, because of this I'm passing the prop `errorsByLanguage` to `LocalizedTextField` which in turn gets passed to `LocalizedTextInput` As far as I can see in the code [here](https://github.com/commercetools/ui-kit/blob/main/packages/components/fields/localized-text-field/src/localized-text-field.tsx#L296) which is used [here](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/localized-text-input/src/localized-text-input.tsx#L347)

As you can see from the the code, the error per language is being rendered as it is meaning, not allowing translation or even rendered within `ErrorMessage` to show it in red color. because of this I think this input should change the prop requirements to allow passing nodes.

**Expected behavior**
Because the errors are rendered as is, I expect to be able to pass a node to allow me to pass my own custom component (without facing prop validation errors) which would allow me to add translations to my errors and also have control over the color options (red color mostly)

**Screenshots**
image

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Improve handling styles for errors/warnings in localized inputs and fields.

fixes #2730

## Description

When localized inputs and fields components users want to provide some error or warning messages, they are currently required to provide a `ReactNode` element using one of our `ErrorMessage` or `WarningMessage` components to wrap the text in order to get the rights styles.

Example:

```jsx
<LocalizedTextField
  title="Title"
  value={value}
  onChange={handleChange}
  errorsByLanguage={{
    en: <ErrorMessage>This is an error</ErrorMessage>,
  }}
/>
```

This should not be a consumer's responsibility so we're changing the way these components handle `errors` and `warnings` so they will include the right styles by default.

Example:
```jsx
<LocalizedTextField
  title="Title"
  value={value}
  onChange={handleChange}
  errorsByLanguage={{
    en: 'This is an error',
  }}
/>
```

We still support the former, but we encourage using the new way to provide errors and warnings.

### CODE REVIEW

**Review by kark — COMMENTED** [review]
Thanks for this improvement 💯 looking good

File: packages/components/inputs/localized-money-input/src/localized-money-input.story.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ storiesOf('Components|Inputs', module)
                 Object.values(errors).some((error) => error.length > 0)
                   ? Object.entries(errors).reduce((acc, [currency, error]) => {
                       if (error.length === 0) return acc;
-                      acc[currency] = <ErrorMessage>{error}</ErrorMessage>;
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply packages/components/inputs/localized-money-input/src/localized-money-input.story.js:89]: Thanks Kacper 👍 

Updated here: 0b92edcf58ee71e246571242e72dc97f7bb2ad29

File: .changeset/beige-mails-explain.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+With the new update, you no longer need to wrap the error text so you can provide it as it is:
+
+```jsx
+<LocalizexTextInput
+  title="Title"
+  value={value}
+  onChange={handleChange}
+  warnings={{
+    en: 'This is a warning</WarningMessage',
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply .changeset/beige-mails-explain.md]: Thanks 👍 

Updated here: 6d4e3d917c9f63ea196596f3c4f0fe73e5274910

File: .changeset/beige-mails-explain.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+'@commercetools-uikit/localized-multiline-text-input': minor
+'@commercetools-uikit/localized-rich-text-input': minor
+'@commercetools-uikit/localized-money-input': minor
+'@commercetools-uikit/localized-text-input': minor
+---
+
+---
+'@commercetools-uikit/localized-multiline-text-field': patch
+'@commercetools-uikit/localized-text-field': patch
+---
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply .changeset/beige-mails-explain.md:11]: I'm sorry but I think I don't understand this one.

There are two changesets added: one is for the field components and the other one is for the input ones.
I think they are not repeated.

Can you maybe double check?

[the remark this answers is not in the document]
Comment (kark) [reply .changeset/beige-mails-explain.md:11]: I think they are. Please look at the contents here:
```md
---
'@commercetools-uikit/localized-multiline-text-input': minor
'@commercetools-uikit/localized-rich-text-input': minor
'@commercetools-uikit/localized-money-input': minor
'@commercetools-uikit/localized-text-input': minor
---

---
'@commercetools-uikit/localized-multiline-text-field': patch
'@commercetools-uikit/localized-text-field': patch
---
```
While the other one:
```md
---
'@commercetools-uikit/localized-multiline-text-field': minor
'@commercetools-uikit/localized-text-field': minor
---
```

**Review by ddouglasz — APPROVED** [review]
Looks good, thank you  👏

**Review by kark — APPROVED** [review]
🎉

**Pull request merged**